AutoGrade/admin.py

```diff
@@ -97,11 +111,11 @@ def get_queryset(self, request):
         if request.user.is_superuser:
             return qs
         return qs.filter(course__instructor__user=request.user)
-    
+
 @admin.register(Submission) 
 class SubmissionModelAdmin(admin.ModelAdmin):
     def get_queryset(self, request):
         qs = super(SubmissionModelAdmin, self).get_queryset(request)
         if request.user.is_superuser:
             return qs
-        return qs.filter(assignment__course__instructor__user=request.user)
+        return qs.filter(assignment__course__instructor__user=request.user)
```

Comment:
Your editor is probably removing the NL character from the the last line. I believe this is not good. (Not an issue for this PR but you should fix this. Just a nitpick =] )

Reply:
Acknowledged.